src/test/resources/freemarkerTemplates/ISOToMT940.ftl

```diff
@@ -0,0 +1,259 @@
+					<#else>
+						<#assign row = row + ["PREF/" + detail["C:Refs/C:PmtInfId"]] />
+						<#assign rowUpdated = true>
+					</#if>
+				</#if>
+				<#assign row = row + ["\n"] />
+			</#list>
+			<#assign entryDetails = entryDetails + [row] />
+		<#else>
+			<#assign rowUpdated = false>
+			<#if entry["C:NtryDtls/C:TxDtls/C:Refs/C:EndToEndId"]?has_content>
+				<#if rowUpdated>
+					<#assign row = row + ["/EREF/" + entry["C:NtryDtls/C:TxDtls/C:Refs/C:EndToEndId"]] />
+				<#else>
+					<#assign row = row + ["EREF/" + entry["C:NtryDtls/C:TxDtls/C:Refs/C:EndToEndId"]] />
+					<#assign rowUpdated = true>
+				</#if>
+			</#if>
+			<#if entry["C:NtryDtls/C:TxDtls/C:Refs/C:InstrId"]?has_content>
+				<#if rowUpdated>
+					<#assign row = row + ["/IREF/" + entry["C:NtryDtls/C:TxDtls/C:Refs/C:InstrId"]] />
+				<#else>
+					<#assign row = row + ["IREF/" + entry["C:NtryDtls/C:TxDtls/C:Refs/C:InstrId"]] />
+					<#assign rowUpdated = true>
+				</#if>
+			</#if>
+			<#if entry["C:NtryDtls/C:TxDtls/C:Refs/C:PmtInfId"]?has_content>
+				<#if rowUpdated>
+					<#assign row = row + ["/PREF/" + entry["C:NtryDtls/C:TxDtls/C:Refs/C:PmtInfId"]] />
+				<#else>
+					<#assign row = row + ["PREF/" + entry["C:NtryDtls/C:TxDtls/C:Refs/C:PmtInfId"]] />
+					<#assign rowUpdated = true>
+				</#if>
+			</#if>
+			<#assign row = row + ["\n"] />
+			<#assign entryDetails = entryDetails + [row] />
+		</#if>
+	</#if>
+</#if>
+:86:<#list entryDetails as row><#assign firstElement=true/><#list row as element><#if !firstElement>/</#if>${element}</#list></#list><#if entry["C:AddtlNtryInf"]?has_content><#if !firstElement>/</#if>${entry["C:AddtlNtryInf"] + "\n"}</#if></#if></#list><#if firstStat>:62F:<#else>:62M:</#if><#if closingBal["C:CdtDbtInd"]=="CRDT">C<#else>D</#if><#if closingBal["C:Dt/C:DtTm"]?has_content><@dateTimeFormat dateTimeInd=true input=closingBal["C:Dt/C:DtTm"] format="yyMMdd"/><#else><@dateTimeFormat dateTimeInd=false input=closingBal["C:Dt/C:Dt"] format="yyMMdd"/></#if>${closingBal["C:Amt/@Ccy"]}${closingBal["C:Amt"]?split(".")[0]},${closingBal["C:Amt"]?split(".")[1]}
+<#if closingAvlBal??>
+:64:<#if closingAvlBal["C:CdtDbtInd"]=="CRDT">C<#else>D</#if><#if closingAvlBal["C:Dt/C:DtTm"]?has_content><@dateTimeFormat dateTimeInd=true input=closingAvlBal["C:Dt/C:DtTm"] format="yyMMdd"/><#else><@dateTimeFormat dateTimeInd=false input=closingAvlBal["C:Dt/C:Dt"] format="yyMMdd"/></#if>${closingAvlBal["C:Amt/@Ccy"]}${closingAvlBal["C:Amt"]?split(".")[0]},${closingAvlBal["C:Amt"]?split(".")[1]}
+</#if>
+<#if forwardAvlFunds??>
+:65:<#if forwardAvlFunds["C:CdtDbtInd"]=="CRDT">C<#else>D</#if><#if forwardAvlFunds["C:Dt/C:DtTm"]?has_content><@dateTimeFormat dateTimeInd=true input=forwardAvlFunds["C:Dt/C:DtTm"] format="yyMMdd"/><#else><@dateTimeFormat dateTimeInd=false input=forwardAvlFunds["C:Dt/C:Dt"] format="yyMMdd"/></#if>${forwardAvlFunds["C:Amt/@Ccy"]}${forwardAvlFunds["C:Amt"]?split(".")[0]},${forwardAvlFunds["C:Amt"]?split(".")[1]}
+</#if>
+-}
```

Comment:
Add EoF line

Reply:
Added in a335dc1f39d3a52600c391d8b6daf8093be6f42c